backend/event/models.py

```diff
@@ -61,13 +60,19 @@ class LetterAction(models.Model):
         help_text="epistolary events this letter action belongs to",
     )
 
+    gifts = models.ManyToManyField(
+        to=Gift,
+        related_name="letter_actions",
+        help_text="Gifts associated to this letter action",
+    )
+
     def __str__(self):
         categories = self.categories.all()
         category_names = [category.get_value_display()
                           for category in categories]
         category_desc = ", ".join(category_names)
         letters = ", ".join(letter.__str__() for letter in self.letters.all())
-        return f"{category_desc} of {letters}"
+        return f"{category_desc} of {letters} ({self.date.display_date})"
```

Comment:
If it's useful for the admin overview, we could also add the date to the admin list display.

Reply:
I'm interested, but I'm not sure what you mean. Would this be a dynamic label like the date ranges you have added to the Letter admin?

Reply:
Ah, I meant that you could configure the admin class of this model with something like `list_display = ['description', 'display_date']` (if `description` and `display_date` are database fields or properties with `@admin.display()` decorators). Then the admin would show the date as a second column in the overview.